app/domain/errors.rb

```diff
@@ -401,6 +401,18 @@ module AuthnJwt
               "   with field in the decoded token that contains the identity",
         code: "CONJ00075E"
       )
+
+      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
```

Comment:
@shulifink please review this messages

CC : @hilagross

Reply:
I think this is too much details for an error msg
maybe we should keep the first sentence and add a troubleshooting section with the error code, like we are doing in [GCP authn doc page](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Operations/Services/cjr-gcp-authn.htm?tocpath=Fundamentals%7CAuthentication%7C_____7#TroubleshootingtheGCPAuthenticator)

@shulifink  WDYT?